### ISSUE TITLE
0.23.1-SNAPSHOT Docker image: ERROR io.zeebe.broker.system - Unexpected error on stream processor installation!

### ISSUE THREAD

**Issue by jwulf** [zeebe-io/zeebe#4433]
Starting the 0.23.1-SNAPSHOT image from Docker Hub with no mounted volumes, the following message is emitted in the logs: 

```
zeebe_broker | 2020-05-02 12:16:33.042 [Broker-0-ZeebePartition-1] [Broker-0-zb-actors-1] ERROR io.zeebe.broker.system - Unexpected error on stream processor installation!
zeebe_broker | java.lang.RuntimeException: Could not deserialize object. Deserialization stuck at offset 1 of length 8
zeebe_broker |  at io.zeebe.msgpack.UnpackedObject.wrap(UnpackedObject.java:35) ~[zeebe-msgpack-value-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.db.impl.rocksdb.transaction.TransactionalColumnFamily.get(TransactionalColumnFamily.java:66) ~[zeebe-db-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.db.impl.rocksdb.transaction.TransactionalColumnFamily.get(TransactionalColumnFamily.java:134) ~[zeebe-db-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.db.impl.rocksdb.transaction.TransactionalColumnFamily.get(TransactionalColumnFamily.java:58) ~[zeebe-db-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.engine.state.LastProcessedPositionState.getPosition(LastProcessedPositionState.java:33) ~[zeebe-workflow-engine-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.engine.state.ZeebeState.getLastSuccessfulProcessedRecordPosition(ZeebeState.java:157) ~[zeebe-workflow-engine-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.engine.processor.StreamProcessor.recoverFromSnapshot(StreamProcessor.java:241) ~[zeebe-workflow-engine-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.engine.processor.StreamProcessor.onActorStarted(StreamProcessor.java:99) ~[zeebe-workflow-engine-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:73) [zeebe-util-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:115) [zeebe-util-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.util.sched.ActorThread.run(ActorThread.java:195) [zeebe-util-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker | Caused by: io.zeebe.msgpack.spec.MsgpackReaderException: Unable to determine map type, found unknown header byte 0x00 at reader offset 0
zeebe_broker |  at io.zeebe.msgpack.spec.MsgPackReader.exceptionOnUnknownHeader(MsgPackReader.java:474) ~[zeebe-msgpack-core-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.msgpack.spec.MsgPackReader.readMapHeader(MsgPackReader.java:79) ~[zeebe-msgpack-core-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.msgpack.value.ObjectValue.read(ObjectValue.java:88) ~[zeebe-msgpack-value-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  at io.zeebe.msgpack.UnpackedObject.wrap(UnpackedObject.java:31) ~[zeebe-msgpack-value-0.23.1-SNAPSHOT.jar:0.23.1-SNAPSHOT]
zeebe_broker |  ... 13 more
zeebe_broker | 2020-05-02 12:16:33.043 [Broker-0-StreamProcessor-1] [Broker-0-zb-actors-0] ERROR io.zeebe.logstreams - Actor Broker-0-StreamProcessor-1 failed in phase FAILED.
zeebe_broker | java.lang.RuntimeException: Could not deserialize object. Deserialization stuck at offset 1 of length 8
```

**Comment by jwulf** [comment]
`latest` works with no error. Probably needs to be tagged as 0.23.1.

**Comment by npepinpe** [comment]
@menski please close after deleting the image.

**Comment by menski** [comment]
The image `0.23.1-SNAPSHOT` was a tag for a test before we released `0.23.1` and was not intended to be used. I deleted the SNAPSHOT artifact as the release is available.
